### ISSUE TITLE
QOwnNotes displays UML diagrams?

### ISSUE THREAD

**Issue by it19862** [pbek/QOwnNotes#2552]
Is it possible to build and display UML diagrams in QOwnNotes?

**Comment by louwers** [comment]
Yes, you can install the PlantUML script.

**Comment by louwers** [comment]
![image](https://user-images.githubusercontent.com/649392/172254756-2aa75ea1-86b8-421a-a688-cd1da9407501.png)

![image](https://user-images.githubusercontent.com/649392/172254788-2648d448-4f23-4ffb-9651-d8d21c008019.png)

**Comment by louwers** [comment]
This is the script to install:

![image](https://user-images.githubusercontent.com/649392/172254884-114db258-093b-4be7-b54d-1361b4108a79.png)

**Comment by it19862** [comment]
@louwers 
I don't understand how it works.

I have added a plugin.
I created a note `01_sequence-diagram.md `.
I added the code to the note.

Result: There is no diagram.

I expect to see this: https://plantuml.com/ru/sequence-diagram
I have tried several charts individually

Code
```plantuml
@staruml
Alice -> Bob: Authentication Request
Bob --> Alice: Authentication Response

Alice -> Bob: Another authentication Request
Alice <-- Bob: another authentication Response
@enduml

```
![07 06 2022_09-15-01](https://user-images.githubusercontent.com/18245828/172309177-064492f8-5486-47f4-b4ff-241c9f1d8514.png)
![07 06 2022_09-15-48](https://user-images.githubusercontent.com/18245828/172309191-aeedbdd4-c8a8-4c2f-a240-3bed2b9d6963.png)
